### ISSUE TITLE
Cannot see details of the failed contracts after running provider verification test.

### ISSUE THREAD

**Issue by danilchican** [pact-foundation/pact-jvm#1308]
I have experienced with the following behavior when some of the tests failed but in reports/logs I can't see details (body diff, etc) what actually wrong. But some time ago it was working fine. But additionally I would like to add that I need to run debug panel with variables to see what was actually wrong.
Could you investigate it?
![image](https://user-images.githubusercontent.com/13799032/108045481-cc70db00-7043-11eb-865e-520bdbbda85d.png)

**Comment by uglyog** [comment]
What version of Pact-JVM are you using?

**Comment by danilchican** [comment]
@uglyog, `au.com.dius.pact.provider:junit5:4.1.12`

**Comment by uglyog** [comment]
When I run the tests, I definitely see the details in the test report

![image](https://user-images.githubusercontent.com/863666/108583568-657a5b80-738e-11eb-824f-de2aede2afe0.png)

Are you getting this when running from the IDE?

**Comment by danilchican** [comment]
@uglyog , yep. But even in html tests running report there is the same as in IDE. 
I didn't see such examples of error.

**Comment by uglyog** [comment]
Can you try the latest version?

**Comment by danilchican** [comment]
> Can you try the latest version?

Is it was fixed/improved in the latest version?

**Comment by uglyog** [comment]
No, but I'm unable to replicate your problem, so there is not much else I can do.
